Makefile.SH

```diff
@@ -1418,8 +1418,8 @@ _cleaner2:
 	-rmdir lib/Test2/EventFacet lib/Test2/Event/TAP lib/Test2/Event
 	-rmdir lib/Test2/API/InterceptResult lib/Test2/API lib/Test2
 	-rmdir lib/Test/use lib/Test/Tester lib/Test/Builder/Tester
-	-rmdir lib/Test/Builder/IO lib/Test/Builder lib/Test lib/Term
-	-rmdir lib/TAP/Parser/YAMLish lib/TAP/Parser/SourceHandler
+	-rmdir lib/Test/Builder/IO lib/Test/Builder lib/Test lib/Term/Table
+	-rmdir lib/Term lib/TAP/Parser/YAMLish lib/TAP/Parser/SourceHandler
```

Comment:
Shouldn't this be in the win32/ makefile too?

Reply:
Yes almost certainly. All of the edits to any files here were made automatically by the `sync-with-cpan` tool. It's possible that has missed some things. I'll have a look and see if I can guess around it manually.

Reply:
Huh, interesting. I don't know why, but both the `win32` Makefile templates contain just a list of the toplevel lib dir directories, not every path inside. Since they already had `$(LIBDIR)\Term` and `$(LIBDIR)\Test2` in the list, it doesn't appear that anything new needs adding there.